Engines/Wine/Verbs/D9VK/script.js

```diff
@@ -115,8 +113,10 @@ class D9VK {
         wine.wizard(wizard);
         wine.prefix(container);
 
-        const versions = getGithubReleases("Joshua-Ashton", "d9vk", wizard);
-        const selectedVersion = wizard.menu(tr("Please select the version."), versions, versions[0]);
+        const versions = fetchGithubReleases("Joshua-Ashton", "d9vk", wizard);
+        const versionStrings = extractGithubReleaseStrings(versions);
+
+        const selectedVersion = wizard.menu(tr("Please select the version."), versionStrings, versionStrings[0]);
 
         // install selected version
         new D9VK(wine).withVersion(selectedVersion.text).go();
```

Comment:
Should we continue to pass a `string` as the version to the verb or should we pass a `version` object?
An disadvantage of passing a `version` object is that it is github specific.

Reply:
`string`. Otherwise you can never write something like `withVersion("2.3")`.

Reply:
The issue with passing a string is that it is misleading in my opinion. The reason is that it is unclear what the version string for a github release is. Is it `tag_name`, `name`, `id` or some other value of the version?

Reply:
That should be internal knowledge to the verb. The user should not have to bother about it (when calling `withVersion`).